### DISCUSSION TITLE
Simulator Speed

### DISCUSSION THREAD

**Discussion by Dilcan44** [carla-simulator/carla#7031] [General]
Is it normal that once i execute the "generate_traffic" example all the simulator velocity changes? It results in all the cars and pedestrians going at least twice as fast as they should

**Comment by TimeExplorerK** [comment]
No clue if it's normal, but mine does the same. Then, if I try to run manual_control.py (included in package) the pygame window lags by a significant amount, with the lag increasing over time. Interestingly, Carla.exe itself shows the manually controlled vehicle's motions on time.

**Reply by Dilcan44** [reply]
I tried changing the "delta" parameter on "generate_traffic" and it slowed the traffic speed. Maybe you should try it, i hope it doesn't affect on other things.
